inference-engine/tools/vpu/vpu_compile/main.cpp

```diff
@@ -25,6 +25,13 @@ static constexpr char model_message[] = "Required. Path to xml model.";
 static constexpr char plugin_path_message[] = "Optional. Path to a plugin folder.";
 static constexpr char output_message[] = "Optional. Path to the output file. Default value: \"<model_xml_file>.blob\".";
 static constexpr char config_message[] = "Optional. Path to the configuration file. Default value: \"config\".";
+static constexpr char platform_message[] = "Optional. Specifies movidius platform."
+                                           " Supported values: VPU_MYRIAD_2450, VPU_MYRIAD_2480."
+                                           " Overwrites value from config.\n"
+"                                             This option must be used in order to compile blob"
+                                           " without a connected Myriad device.\n"
+"                                             PLATFORM configuration keys and values ​​are deprecated"
+                                           " and may not be supported in future releases.";
```

Comment:
Do not return `MY2` back to life there. My opinion here since there is no valid options except Myriad-X - tool should never require a device.

@taka-no-me

Reply:
Earlier, you told me that we were a little rushed to remove this code from the vpu_compile. What's your opinion now?
As I understand it, you mean to make it so that the model is only compiled as if myriadx is connected to the host. Right?

Reply:
As I understand it, Gleb meant that "tool should never require a device" = "tool should compile the model even without using this option, since there is only one valid value (only MYX)".
Or not?

Reply:
No I mean vice versa. Remove option from tool and work always even if there is no device.
My opinion before (rushed to remove this code) was about losing possibility to compile without device, but now since we have only 2480 left I think we can remove option if tool always behave as it was set to `VPU_MYRIAD_2480`

> Earlier, you told me that we were a little rushed to remove this code from the vpu_compile. What's your opinion now?
> As I understand it, you mean to make it so that the model is only compiled as if myriadx is connected to the host. Right?

Reply:
Can you take a look now?
I removed option and used PLATFORM configuration key, filled it with the only possible value, so myriad_compile works anyway, even w/o device.

Reply:
@Maxim-Doronin  @gladilov-gleb 
Could you take another look?